library/MedEx/API.php

```diff
@@ -2058,7 +2069,8 @@ public function login() {
         $this->curl->setData(array(
```

Comment:
any particular special cURL configuration stuff needed server-side?
I suppose the install needs to check for:
extension=php_curl.dll in phpinfo